s3tests_boto3/readwrite.py

```diff
@@ -57,7 +57,7 @@ def reader(bucket, worker_id, file_names, queue, rand):
                         traceback=traceback.format_exc(),
                         ),
                     )
-                print "ERROR:", m
+                print("ERROR:", m)
```

Comment:
This requires future python feature enabled in order it prints the same way as in py2